packages/packagers/js/src/dev-prelude.js

```diff
@@ -73,7 +73,7 @@
         localRequire,
         module,
         module.exports,
-        this
+        globalObject
```

Comment:
Apparently we relied on `(function(){ console.log(this === globalThis); })()`, even though we already have the proper global object available